packages/components/src/theme/theme-dark/theme-dark-semantic-chart.css

```diff
@@ -0,0 +1,52 @@
+/* stylelint-disable alpha-value-notation */
+/* stylelint-disable custom-property-empty-line-before */
+:root {
+  --dh-color-chart-bg: var(--dh-color-content-background);
+  --dh-color-chart-plot-bg: var(--dh-color-gray-200);
+  --dh-color-chart-title: var(--dh-color-white);
+
+  /* Colorway */
+  --dh-color-chart-colorway-1: var(--dh-color-visual-cyan);
+  --dh-color-chart-colorway-2: var(--dh-color-visual-green);
+  --dh-color-chart-colorway-3: var(--dh-color-visual-yellow);
+  --dh-color-chart-colorway-4: var(--dh-color-visual-purple);
+  --dh-color-chart-colorway-5: var(--dh-color-visual-orange);
+  --dh-color-chart-colorway-6: var(--dh-color-visual-red);
+  --dh-color-chart-colorway-7: var(--dh-color-visual-chartreuse);
+  --dh-color-chart-colorway-8: var(--dh-color-visual-fuchsia);
+  --dh-color-chart-colorway-9: var(--dh-color-visual-seafoam);
+  --dh-color-chart-colorway-10: var(--dh-color-visual-magenta);
+  --dh-color-chart-colorway-11: var(--dh-color-white);
```

Comment:
`colorway` should just be one variable - they could have less or more than 11 items in the colorway.